Tests/FoundationEssentialsTests/DataLegacyABITests.swift

```diff
@@ -0,0 +1,24 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if FOUNDATION_FRAMEWORK
+@_spi(FoundationLegacyABI) @testable import Foundation
+import Testing
+
+@Test func validateLegacyABI() {
```

Comment:
Normally we try to organize our tests into test suites - is it possible to do that here (I suspect there will be other legacy ABI tests that we may add in the future so it would be great to have a home for that here